unstructured/partition/ocr.py

```diff
@@ -385,84 +450,86 @@ def zoom_image(image: PILImage, zoom: float = 1) -> PILImage:
     return PILImage.fromarray(new_image)
 
 
-def get_ocr_layout_from_image(
+def get_ocr_data_from_image(
     image: PILImage,
     ocr_languages: str = "eng",
-    ocr_agent: str = "tesseract",
-) -> List[TextRegion]:
+    ocr_agent: str = OCR_AGENT_TESSERACT,
+    output_type: OCROutputType = OCROutputType.STRING,
+) -> Union[str, List[TextRegion]]:
```

Comment:
In my experience a function that takes an input that specifies what format the output takes is an antipattern. I'd recommend the following:
1. Create another function (maybe named `get_ocr_text_from_image`) that handles `PILImage -> str`
2. Within both functions, spin the agent dependent code out into their own functions, so that `get_ocr_data_from_image` and it's `str` sibling are just routing functions. This already seems to be mostly done with the `parse_ocr_data_{agent}` functions, but the Tesseract preprocessing code is IMO out of place here and should live in a Tesseract-specific function. Maybe directly in `parse_ocr_data_tesseract`?

Reply:
@qued I combined `get_ocr_layout_from_image()` and `get_ocr_text_from_image()` to `get_ocr_data_from_image()` in this PR, following the same pattern as the `unstructured_pytesseract.image_to_data()`.
```
ocr_df = unstructured_pytesseract.image_to_data(
    np.array(image),
    lang=ocr_languages,
    output_type=Output.DATAFRAME,
)
```
Do you recommend to revert it back?